tes-adapter/src/main/java/com/epam/pipeline/tesadapter/controller/TesAdapterController.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.pipeline.tesadapter.controller;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class TesAdapterController {
+
+    @GetMapping("/v1/tasks/service-info")
+    public ResponseEntity<String> serviceInfo(
+            @RequestParam(name = "status", required = false, defaultValue = "OK") String status){
+        if (status.equalsIgnoreCase("ok")){
+            return new ResponseEntity<>("The system is fully operational!", HttpStatus.OK);
+        }
+        return new ResponseEntity<>("The system is not operable!", HttpStatus.NOT_FOUND);
```

Comment:
btw when you want to say that here is some problem with a server it's better to return Http Code like 5xx (where x  is digit from 0 to 9) because 5xx codes relate to server problem, and  `HttpStatus.NOT_FOUND` which is actually `404`, means that server works and just can't find what you are asking for 
here some information about http codes: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status

Reply:
fixed